### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this activity I continued working on the Leaderboard list app, sending and receiving data from/to the Leaderboard API. For these purposes, I followed below steps:

1. Read the Leaderboard API documentation to learn how to use this API.
2. Create a new game with the name of your choice by using the API.
3. Implement the "Refresh" button.
4. Implement the form "Submit" button.
5. Use async and await JavaScript features to consume the API.
6. No styling is required.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+function addToUI(arr) {
+  list.innerHTML = '';
+  arr.forEach((el) => {
+    list.innerHTML += `
+    <li class="item">${el.user} : ${el.score}</li>
+    `;
+  });
+}
+
```
Comment (dasileker) [inline src/index.js]: - [x] Please use the arrow function. 👉 see here [requirement arrow function](https://user-images.githubusercontent.com/61562626/127013730-08f3b34e-e360-4b80-869b-7e83c7c09340.jpeg)

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hello @tacettinkustu 🖐🖐** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
Please check the comments under the review.

- [x] Ensure you use **`arrow functions`** rather than the **`function`** keyword. 

- Check this link about [ES6 Arrow function](https://javascript.info/arrow-functions-basics)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (tacettinkustu) [reply src/index.js]: I changed my functions

**Review by carlos-ssh — APPROVED** [review]
Hi @tacettinkustu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**